### PR TITLE
'Admin Page Tutorial' improvements

### PR DESCRIPTION
**Pull request by mav1n**
- Step 3: 'Print QR Code' added in Admin Page Tour
- Added done icon as next step icon in the last step of the tour.

### CODE REVIEW

File: simplq/src/components/common/Tour/Tour.jsx [hunk 1]
```diff
@@ -24,9 +26,9 @@ export default (props) => {
...
         leftArrow.childNodes[0].style.color = 'grey';
         rightArrow.childNodes[0].style.color = 'white';
-      } else if (stepNumber === props.toursteps.length - 1) {
+      } else {
         leftArrow.childNodes[0].style.color = 'white';
-        rightArrow.childNodes[0].style.color = 'grey';
+        rightArrow.childNodes[0].style.color = 'white';
       }
     } else {
       Sentry.captureMessage('left-arrow or right-arrow selectors of reatTour package not found');
```
Comment (daltonfury42) [inline simplq/src/components/common/Tour/Tour.jsx:34]: Let us remove this entire function. Looks hacky, and the default behaviour looks good to me.

**Review by daltonfury42 — APPROVED** [review]
Thanks for the PR!

**Review by maaverik — APPROVED** [review]
Thanks for the PR. Please see if you can modify it according to @daltonfury42's comments.

File: simplq/src/components/common/Tour/Tour.jsx (hunk 1 above)
Comment (mav1n) [reply simplq/src/components/common/Tour/Tour.jsx:34]: image

In the default behaviour, the active 'next' icon is more dull than the disabled 'prev' icon. Shouldn't it be the opposite?

File: simplq/src/components/common/Popup/QrCode.jsx [hunk 2]
```diff
@@ -71,7 +71,7 @@ export const QrCode = ({ queueName, handleModalClose }) => {
         ref={componentPrintRef}
       />
       <div className={styles['actionContainer']}>
-        <StandardButton onClick={handlePrint} icon={<PrintIcon />}>
+        <StandardButton tourTag={tourTag} onClick={handlePrint} icon={<PrintIcon />}>
```
[the remark this answers is not in the document]
Comment (mav1n) [reply simplq/src/components/common/Popup/QrCode.jsx:74]: It should work. I'll check.

File: simplq/src/components/common/Tour/Tour.jsx (hunk 1 above)
Comment (maaverik) [reply simplq/src/components/common/Tour/Tour.jsx:34]: Yes, it should be the opposite. In that case, this function will have to stay unless there's a better possible implementation.

Comment (daltonfury42) [reply simplq/src/components/common/Tour/Tour.jsx]: Yeah, I tried locally. 

Right now it's a hacky implementation, and if you go back from the last tutorial, the arrow highlighting is not correct. I would prefer the fix the root cause than fix a bug on top of a hacky implementation.

@maaverik can we remove this function and fix the color as a separate PR?

File: simplq/src/components/common/Popup/QrCode.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (mav1n) [reply simplq/src/components/common/Popup/QrCode.jsx]: Updated this in the PR.

File: simplq/src/components/common/Tour/Tour.jsx (hunk 1 above)
Comment (mav1n) [reply simplq/src/components/common/Tour/Tour.jsx]: I improved the implementation and fixed the arrow highlighting in the same PR.

Comment (daltonfury42) [reply simplq/src/components/common/Tour/Tour.jsx]: @mav1n has handled this elegantly in this PR itself. :)

**Pull request merged**